pkg/aws/ec2/api/helper.go

```diff
@@ -69,8 +69,8 @@ func NewEC2APIHelper(ec2Wrapper EC2Wrapper, clusterName string) EC2APIHelper {
 	// Set the key and value of the cluster name tag which will be used to tag all the network interfaces created by
 	// the controller
 	clusterNameTag = ec2types.Tag{
-		Key:   aws.String(fmt.Sprintf(config.ClusterNameTagKeyFormat, clusterName)),
```

Comment:
This won't work for VPC RC's ENI, will it?

Reply:
Since tag filters are ORed, I think we need both?

Reply:
I am not certain in this case since keys are diff. But we can test. Let's keep them backwards compatible.